### PR TITLE
Assert on `payouts` in `transfer.test.ts`

### LINKED ISSUE #1458: Test effects of `transfer`
Current test of `transfer` makes no assurance that assets actually flow to intended recipients - marked as a [todo](https://github.com/statechannels/go-nitro/blob/8c3578087e9c5fcb9110beb48dbce814d9df9f7c/nitro-protocol/test/contracts/MultiAssetHolder/transfer.test.ts#L196).

### PR DESCRIPTION
**Pull request by NiloCK**
PR adds checks that payouts are executed as expected in `transfer.test.ts`

Closes #1458

### CODE REVIEW

**Review by lalexgap — DISMISSED** [review]

**Comment by geoknee** [comment]
Maybe I'm missing something, but I don't think this test does any assertions with the `payouts` variable?

File: nitro-protocol/src/helpers.ts [hunk 1]
```diff
@@ -73,3 +73,20 @@ export function getRandomNonce(seed: string): string {
...
+export function isExternal(destination: string): boolean {
+  if (destination.substring(0, 2) != '0x' || destination.length != 66) {
+    return false;
+  }
+  // check the leading 24 zeros. eg:
+  // 0x000000000000000000000000a5aeef8f28edd5a55554856f107d4cc054d41b23
+  console.log(`destination: ${destination}`);
+  console.log(destination.substring(2, 26));
+  return destination.substring(2, 26) == ''.padEnd(24, '0');
+}
```
Comment (geoknee) [inline nitro-protocol/src/helpers.ts:92]: There's already a helper for this here https://github.com/statechannels/go-nitro/blob/2b0bc7c5998abe021970704f448d030844a05928/nitro-protocol/src/contract/channel.ts#L11

Comment (geoknee) [reply nitro-protocol/src/helpers.ts]: [4cbeec0](https://github.com/statechannels/go-nitro/pull/1602/commits/4cbeec0ad46a82779ccc3045285f84857d438ad0)

Comment (NiloCK) [reply nitro-protocol/src/helpers.ts]: :+1: I was sure there had to be, but my ctrl+f skills were failing me.

**Comment by geoknee** [comment]
Can't seem to get the RPC client test to pass on this branch #1613

**Review by geoknee — APPROVED** [review]

**Pull request merged**